base/Makefile

```diff
@@ -322,55 +255,16 @@ ${CLUSTER_GEN}/remote-healthcheck-image: ${CLUSTER_GEN}/healthcheck-image-pushed
 	kubectl patch ServiceAccount default --patch "automountServiceAccountToken: false" >&2
 	mkdir -p ${CLUSTER_GEN} >&2
 
-.FORCE:
```

Comment:
hmm, I don't understand how this works if the .FORCE target is not there anymore. I.e. it's still used by the targets above?

Reply:
I forgot to delete it in the other targets, but, no, it's not used anymore. I will make a commit deleting it in the other targets.

Reply:
what changed that it's not needed anymore?
For example in:
```Makefile
${CLUSTER_GEN}/remote-image: ${CLUSTER_GEN}/image-pushed .FORCE
```
If we remove the .FORCE, it might use a stale remote-image file if I'm not mistaken

Reply:
added back

Reply:
i didn't know what it was for